.travis.yml

```diff
@@ -71,7 +71,18 @@ jobs:
         - true
       script:
         - ./run-checks --spread-no-ubuntu
-env:
+    - stage: integration
+      name: Unstable systems
+      dist: xenial
+      addons:
+        apt:
+          packages:
+          - xdelta3
+      install:
+        # override the default install for language:go
+        - true
+      script:
+        - ./run-checks --spread-unstable || true
```

Comment:
With that we always see "green" as the result, no? I wonder if there is a way in travis to somehow flag if something needs attention or if travis is really only binary (failed xor succeed).

Reply:
Maybe https://docs.travis-ci.com/user/customizing-the-build/#rows-that-are-allowed-to-fail is helpful here?

Reply:
@mvo5 change done, tests running right now, thanks for the suggestion!!